tests/materials/material.mutation.test.js

```diff
@@ -293,31 +247,6 @@ describe('material mutations tests', () => {
     expect(updatedMaterial).toHaveProperty('message', MATERIAL_NOT_FOUND);
   });
 
-  it('should return error when update material with already existing name will ', async () => {
```

Comment:
We throw this error in our services, so why you remove this?
![image](https://user-images.githubusercontent.com/59802190/105381223-7319b580-5c17-11eb-8fbb-c10c64375009.png)